commit 803f475286c861cb5e7c9fa4f4dfadc5d0900177
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1331690354 -0500

    Remove "More Applications" from TDE menu. Add Utility category to KEdit.
    This partially resolves bug report 653.

--- a/kdf/kdf.desktop
+++ b/kdf/kdf.desktop
@@ -85,4 +85,4 @@ Terminal=false
 DocPath=kdf/index.html
 X-KDE-StartupNotify=true
 X-DCOP-ServiceType=Multi
-Categories=Qt;KDE;System;X-KDE-More;
+Categories=Qt;KDE;System;
diff --git a/kdf/kwikdisk.desktop b/kdf/kwikdisk.desktop
index 28ca5b5..c723364 100644
--- a/kdf/kwikdisk.desktop
+++ b/kdf/kwikdisk.desktop
@@ -74,5 +74,5 @@ GenericName[zh_CN]=可移动介质工具
 GenericName[zh_TW]=可移動媒體公用程式
 GenericName[zu]=Umsebenzi Wonozindaba Ogudluzekayo
 Terminal=false
-Categories=Qt;KDE;System;X-KDE-More;
+Categories=Qt;KDE;System;
 OnlyShowIn=KDE;
diff --git a/kedit/KEdit.desktop b/kedit/KEdit.desktop
index f4e3abc..0170151 100644
--- a/kedit/KEdit.desktop
+++ b/kedit/KEdit.desktop
@@ -79,4 +79,4 @@ Name[xh]=Abahleli Be K
 Name[zh_TW]=KDE 編輯器
 X-KDE-StartupNotify=true
 X-DCOP-ServiceType=Multi
-Categories=Qt;KDE;TextEditor;
+Categories=Qt;KDE;Utility;TextEditor;
diff --git a/ktimer/ktimer.desktop b/ktimer/ktimer.desktop
index d34c43b..1d12283 100644
--- a/ktimer/ktimer.desktop
+++ b/ktimer/ktimer.desktop
@@ -60,4 +60,4 @@ Name[xh]=Umjongi Wexesha
 Name[zh_TW]=KDE 計時器
 X-KDE-StartupNotify=true
 X-DCOP-ServiceType=Multi
-Categories=Qt;KDE;Utility;X-KDE-More;
+Categories=Qt;KDE;Utility;
